### PR TITLE
Update changelog

### LINKED ISSUE #2308: Issue warning if an inline assembly instruction shadows a Solidity variable

### PR DESCRIPTION
**Pull request by axic**
Fixes #2308.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
 
 Bugfixes:
- * Code generator: Use ``REVERT`` instead of ``INVALID`` for generated input validation routines.
- * Type Checker: Fix address literals not being treated as compile-time constants.
- * Type Checker: Disallow invoking the same modifier multiple times.
- * Type Checker: Make UTF8-validation a bit more sloppy to include more valid sequences.
+ * Code Generator: Fix ABI encoding of empty literal string.
+ * Code Generator: Fix negative stack size checks.
+ * Code Generator: Use ``REVERT`` instead of ``INVALID`` for generated input validation routines.
+ * Fixed crash concerning non-callable types.
```
Comment (chriseth) [inline Changelog.md]: Please add a component.

File: Changelog.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+ * Fixed segfault with constant function parameters
+ * Inline Assembly: Enforce function arguments when parsing functional instructions.
  * Type Checker: Disallow comparisons between mapping and non-internal function types.
+ * Type Checker: Disallow invoking the same modifier multiple times.
  * Type Checker: Do not treat strings that look like addresses as addresses.
+ * Type Checker: Fix address literals not being treated as compile-time constants.
  * Type Checker: Support valid, but incorrectly rejected UTF-8 sequences.
- * Fixed crash concerning non-callable types.
  * Unused variable warnings no longer issued for variables used inside inline assembly.
```
Comment (chriseth) [inline Changelog.md]: Please add a component.

File: Changelog.md (hunk 1 above)
Comment (axic) [reply Changelog.md]: What was the component?

File: Changelog.md (hunk 2 above)
Comment (chriseth) [reply Changelog.md]: This should be static analyzer.

File: Changelog.md (hunk 1 above)
Comment (chriseth) [reply Changelog.md]: Type checker.

**Comment by axic** [comment]
Please triple check everything is still there :)

**Review by chriseth — APPROVED** [review]

**Pull request merged**